ActionMenu/UIResources/index.js

```diff
@@ -646,7 +646,7 @@ const widget_radial = (function() {
 		$arc.style.clipPath = `polygon(${clip_path})`;
 	}
 
-	const value_label = value => Math.floor(value * 100) + "%";
+	const value_label = value => Math.ceil(value * 100) + "%";
```

Comment:
Ah good catch. I'll need to check, it's odd I used ceil and not round actually. Let me look later.